install/source/index.markdown

```diff
@@ -88,6 +127,6 @@ Its best to contribute to our latest branch, even if you're still on an earlier
     catkin config --extend /opt/ros/${ROS_DISTRO} --cmake-args -DCMAKE_BUILD_TYPE=Release
```

Comment:
```suggestion
    catkin config --extend /opt/ros/${ROS_DISTRO} --cmake-args -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Reply:
We've always advised people to build for release. I'm not convinced this is a good change (what if they really are just deploying this?). Please suggest this change in a separate PR.

Reply:
I reviewed this with a developer in mind. If you're not going to be working on MoveIt, why build from source? The first thing I have to ask people when they post issues about crashing code or other strange things is to build everything with debug symbols enabled.

Why not preempt that sort of situation and have them use `RelWithDebInfo`.

But I'm completely fine with keeping `Release`.